channel/channel.go

```diff
@@ -196,7 +196,7 @@ func (c *Channel) AddSignedState(s state.State, sig state.Signature) bool {
 func (c Channel) AddSignedStates(mapping map[*state.State]state.Signature) bool {
 	allOk := true
 	for state, sig := range mapping {
-		allOk = allOk && c.AddSignedState(*state, sig)
+		allOk = c.AddSignedState(*state, sig) && allOk
```

Comment:
```suggestion
		ok := c.AddSignedState(*state, sig)
		if !ok {
			allOk = false
		}
```
I had written this on the prior PR but discarded it after convincing myself that the existing implementation was correct and sufficiently easy to understand.

I am again convinced that the proposed implementation is correct and sufficiently easy to understand, but I am applying a new personal rule: prefer not to call functions inside larger expressions.

Reply:
Yep. I think naming the return value makes debugging easier, too.